docs/rules/customization.md

```diff
@@ -1,12 +1,32 @@
 # Customization
 
+## Variable Tracing
+The plugin allows a limit back-tracing of variables.
+This will be used to check code like here:
+```js
+const greeting_template = `<p>Hello World!</p>`;
+// ... lots of other code in between ...
+someElemenet.innerHTML = greeting_template;
+```
+
+Currently, backtracing will only allow const and let variables that contain string literals only.
+Further assignments to these variables will also be checked for validation.
```

Comment:
Optional: it would be nice to mention that the order of the assignment statements does not matter, but I'm not sure about how to word it in a non verbose and clear way.